System/Process/Windows.hsc

```diff
@@ -1,4 +1,5 @@
-{-# LANGUAGE CPP #-}
```

Comment:
I'm going to review this file too, but my review will be much less authoritative. Basically: I'm assuming you know what's going on for Windows better than I do, which is a really safe assumption 😄.